code/style.css

```diff
@@ -1,7 +1,258 @@
+body {
```

Comment:
It all looks especially good on mobile 🤩

Reply:
It looks a little jumpy on desktop though 🤔 
https://user-images.githubusercontent.com/51529201/107059026-9dea3900-67d5-11eb-9ecc-20cdda37b8a1.mov

Reply:
I will take a look at this. Thank you for the video :)